api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/ResumoTrimestralDataSetDefinition.java

```diff
@@ -138,4 +143,112 @@ private EptsQuarterlyCohortDefinition getQuarterlyCohort(
     parameters.add(ReportingConstants.LOCATION_PARAMETER);
     return parameters;
   }
+
+  private CohortDefinition getD() {
```

Comment:
@ningosi better move this to [ResumoTrimestralCohortQueries.java](https://github.com/jembi/openmrs-module-eptsreports/blob/develop/api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoTrimestralCohortQueries.java)